presto-orc/src/main/java/com/facebook/presto/orc/reader/LongDirectBatchStreamReader.java

```diff
@@ -186,6 +196,8 @@ private Block readNullBlock(boolean[] isNull, int nonNullCount)
         throw new VerifyError("Unsupported type " + type);
     }
 
+    protected void maybeTransformValues(long[] values, int nextBatchSize) {}
```

Comment:
Why do we need this?

Reply:
Actually I don't think we do here... just took this directly from the IBM code. Will push version without it